[2025-09-30T17:43:01.796+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-30T17:43:01.839+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag3.task_pwd scheduled__2025-09-30T17:41:00+00:00 [queued]>
[2025-09-30T17:43:01.857+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag3.task_pwd scheduled__2025-09-30T17:41:00+00:00 [queued]>
[2025-09-30T17:43:01.858+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-09-30T17:43:01.886+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): task_pwd> on 2025-09-30 17:41:00+00:00
[2025-09-30T17:43:01.904+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=293) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-09-30T17:43:01.909+0000] {standard_task_runner.py:63} INFO - Started process 297 to run task
[2025-09-30T17:43:01.907+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag3', 'task_pwd', 'scheduled__2025-09-30T17:41:00+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/dag3.py', '--cfg-path', '/tmp/tmphcymqpuq']
[2025-09-30T17:43:01.913+0000] {standard_task_runner.py:91} INFO - Job 63: Subtask task_pwd
[2025-09-30T17:43:01.938+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-09-30T17:43:02.037+0000] {task_command.py:426} INFO - Running <TaskInstance: dag3.task_pwd scheduled__2025-09-30T17:41:00+00:00 [running]> on host 7cd44293b4c0
[2025-09-30T17:43:02.266+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag3' AIRFLOW_CTX_TASK_ID='task_pwd' AIRFLOW_CTX_EXECUTION_DATE='2025-09-30T17:41:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-30T17:41:00+00:00'
[2025-09-30T17:43:02.274+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-30T17:43:02.278+0000] {logging_mixin.py:188} INFO - 1
[2025-09-30T17:43:02.279+0000] {logging_mixin.py:188} INFO - 2
[2025-09-30T17:43:02.280+0000] {logging_mixin.py:188} INFO - 3
[2025-09-30T17:43:02.281+0000] {logging_mixin.py:188} INFO - 4
[2025-09-30T17:43:02.281+0000] {logging_mixin.py:188} INFO - 5
[2025-09-30T17:43:02.282+0000] {logging_mixin.py:188} INFO - 6
[2025-09-30T17:43:02.283+0000] {logging_mixin.py:188} INFO - 7
[2025-09-30T17:43:02.283+0000] {logging_mixin.py:188} INFO - 8
[2025-09-30T17:43:02.284+0000] {logging_mixin.py:188} INFO - 9
[2025-09-30T17:43:02.285+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-30T17:43:02.287+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-30T17:43:02.304+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag3, task_id=task_pwd, run_id=scheduled__2025-09-30T17:41:00+00:00, execution_date=20250930T174100, start_date=20250930T174301, end_date=20250930T174302
[2025-09-30T17:43:02.366+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-30T17:43:02.405+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-30T17:43:02.408+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
